python/src/scipp/plotting/figure2d.py

```diff
@@ -81,6 +83,10 @@ def rescale_to_data(self, vmin, vmax):
         """
         Rescale the colorbar limits according to the supplied values.
         """
+        if math.isclose(vmin, vmax):
```

Comment:
Does this also work if the scaling is log?
What does `vmin` and `vmax` give you then? Could you end up making a negative `vmin` by subtracting the offset?

Reply:
No, I am working on fixing some things with log. There are also a couple other issues, such as vmin/vmax getting lost after changing the norm. Will push update soon.